packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -1088,6 +1110,26 @@ function pushLink(
   target: Array<Chunk | PrecomputedChunk>,
   props: Object,
   responseState: ResponseState,
+  textEmbedded: boolean,
+): ReactNodeList {
+  if (enableFloat && resourcesFromLink(props)) {
```

Comment:
This is fine for now to keep these isolated but this is the kind of weird pattern we get from forcing a slice into a separate file like this. This would make more sense inline.

Reply:
yeah, I'll pull more into format config in a future update